src/sensitivities/array.jl

```diff
@@ -1,3 +1,9 @@
 # Let the user get the `size` and `length` of `Node`s.
 Base.size(x::Node, dims...) = size(x.val, dims...)
 Base.length(x::Node) = length(x.val)
+
+# Sensitivity for the first argument of `reshape`.
+import Base.reshape
```

Comment:
Tiny stylistic thing; perhaps put a single `import Base: reshape, size, length` at the top of this file and change `Base.size`->`size` and the same for `length`.

Reply:
Sure. I'll adjust that shortly.